src/stanza.ml

```diff
@@ -13,7 +13,11 @@ let syntax =
     ]
 
 module File_kind = struct
-  type t = Jbuild | Dune
+  type t = Sexp.syntax = Jbuild | Dune
+
+  let of_syntax = function
+    | (0, 0) -> Jbuild
```

Comment:
Even though we are not going to release another version of jbuilder, we should write `(0, _)` here (it's slightly clearer IMO)